lib/ClimaLandSimulations/src/fluxnet_simulation.jl

```diff
@@ -0,0 +1,348 @@
+# note: had to change "$(site_ID)_parameters.jl" k_dry to k_dry_val to avoid a bug, did it for Ozark only for now
+# ERROR: LoadError: cannot assign a value to imported variable Soil.κ_sat_frozen

```

Comment:
I planned to just rename these variables (adding the _val) in this PR, what should I say in an issue?

Reply:
I was suggesting to open an issue documenting the bug you found that explains the error message itself, what causes the error, and maybe an example to reproduce it (something like [this issue](https://github.com/CliMA/ClimaLSM.jl/issues/356)). It's helpful to have issues documenting bugs that come up, so we can see how they were fixed, especially in case they show up again in the future. But if you plan to fix it in this PR and don't think it will come back, it's not a huge deal to go without writing an issue